pyfinviz/test/test_quote.py

```diff
@@ -23,7 +23,7 @@ def test_main_AMZN(self):
         self.assertEqual(20, quote.income_statement_df.columns.size)
         self.assertEqual(8, len(quote.income_statement_df))
         self.assertEqual(11, quote.insider_trading_df.columns.size)
-        self.assertEqual(91, len(quote.insider_trading_df))
+        self.assertTrue(not quote.insider_trading_df.empty) # amount of insider trading can change
```

Comment:
These tests were already failing, so this was just one alternate solution. Let me know if you have any ideas to account for this ever-changing number.